Apps/frontend/src/views/Oscilloscope.svelte

```diff
@@ -120,6 +121,17 @@
         <TimeSweepSlider channel={i} />
       {/each}
     </div>
+    <div class="sliders-wrapper">
+      {#each { length: NUM_CHANNELS } as _, i}
+        <AmplitudeSlider
+          channel={i}
+          type="number"
```

Comment:
I get an error:
> "type' is not defined"

Do we actually need to set the type?

Reply:
works without it. was an artefact left in.